cabal-install/cabal-install.cabal

```diff
@@ -216,14 +217,15 @@ library
         mtl        >= 2.0      && < 2.3,
         network-uri >= 2.6.0.2 && < 2.7,
         pretty     >= 1.1      && < 1.2,
-        process    >= 1.2.3.0  && < 1.7,
+        -- process lower bound moved from Cabal-3.8.1.0 to here:
+        process    >= 1.6.14.0 && < 1.7,
         random     >= 1.2      && < 1.3,
         stm        >= 2.0      && < 2.6,
         tar        >= 0.5.0.3  && < 0.6,
-        time       >= 1.5.0.1  && < 1.12,
+        time       >= 1.5.0.1  && < 1.13,
         zlib       >= 0.5.3    && < 0.7,
         hackage-security >= 0.6.2.0 && < 0.7,
-        text       >= 1.2.3    && < 1.3,
+        text       >= 1.2.3    && < 1.3 || >=2.0 && <2.1,
```

Comment:
I don't remember --- was this tested with 3.8 (as opposed to with master)?

Reply:
I made this revision 2022-08-30.  I strongly suspect that I did test it before revising.
But even if I hadn't, it has now passed the test of time. ;-)

Reply:
Hah, it did. But if so, I bet we tested it on 3.8 (or master before 3.8 has been cut).